lib/Service/SnoozeService.php

```diff
@@ -222,20 +218,14 @@ public function snoozeThreadDB(Message $selectedMessage, int $unixTimestamp, Mai
 			$snooze->setMessageId($message['messageId']);
 			$snooze->setSnoozedUntil($unixTimestamp);
 			$snooze->setSrcMailboxId($srcMailbox->getId());
-			try {
-				$this->messageSnoozeMapper->insert($snooze);
-			} catch(Exception $e) {
-				if ($e->getReason() === Exception::REASON_UNIQUE_CONSTRAINT_VIOLATION) {
-					$messageId = $message['messageId'];
-					if($messageId === null) {
-						throw $e;
-					}
-					$this->messageSnoozeMapper->deleteByMessageIds(array($messageId));
-					$this->messageSnoozeMapper->insert($snooze);
-				} else {
-					throw $e;
-				}
+
+			$messageId = $message['messageId'];
```

Comment:
Why would findMessageIdsByThreadRoot return null for messageId?

Reply:
You're right, messageId shouldn't be null. I just adapted the pattern from the $message->getMessageId(); where the null check was needed.